apps/web/frontend/components/weekTable/weekTableTaskRow.tsx

```diff
@@ -22,7 +24,18 @@ export const WeekTableTaskRow = ({ interval, task }: WeekTableTaskRowProps) => {
           .map((workHour) => workHour.duration)
         const duration = durations.reduce((previous, current) => previous + current, 0)
 
-        return <WeekTableTaskDayCell day={day} taskId={task.id} duration={duration} key={day.toDateString()} />
+        return (
+          <WeekTableTaskDayCell
+            day={day}
+            disabled={
+              (projectStart ? isBefore(day, new Date(projectStart)) : false) ||
+              (projectEnd ? isAfter(day, new Date(projectEnd)) : false)
```

Comment:
You should use `parseISO(...)` from `date-fns`. 

`new Date()` will set the time to 0' a clock Greenwich Mean Time, which is 1' a clock for us.
`parseISO` will set the time to 0' a clock of the timezone of the user, so 0' a clock for us.

If `projectStart` and `day` are set to `2023-01-01`, your code compares:
```
 is '2023-01-01 00:00:00' before '2023-01-01 01:00:00'?
```
This condition is `true` so the input is wrongly disabled.